### PR TITLE
remove deprecation for Ember 2+ Ember.Keys

### PR DESCRIPTION
**Pull request by Blackening999**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,5 +15,6 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+.idea
```
Comment (kellyselden) [inline .gitignore]: I believe this file change isn't relative to this PR

Comment (Blackening999) [inline .gitignore]: no, it's relative and will help with your next contributors. I'm using WebStorm and it's automatically creating some .idea files which become a part of any project.

Comment (kellyselden) [inline .gitignore]: It should be considered in another PR. It doesn't affect the deprecation fix.

Comment (davidgoli) [inline .gitignore]: agreed, any files automatically added by editors are a mistake and won't be merged

**Comment by davidgoli** [comment]
Build failed. Also I don't understand how you would hit `Ember.keys` as the || check defaults to `Object.keys`.

**Comment by davidgoli** [comment]
That said, Object.keys does have very good native support at this time.

**Comment by Blackening999** [comment]
The problem is that it will work with Object.keys for Ember 1.10 as well and the fact I see DEPRECATION in Ember 2.1.0 project, because of useless fallback

**Comment by davidgoli** [comment]
Can you explain why your code would be calling Ember.keys in 2.1.0? It seems like if your browser supports Object.keys you should never see that deprecation warning. But evidently you are.

**Comment by davidgoli** [comment]
The current implementation seems to be the one directly recommended by the Ember core team: https://github.com/emberjs/data/issues/3465

**Comment by Blackening999** [comment]
Well, I don't know why it's so -however I see the DEPRECATION. I agree it's weird cause Object.keys goes first. Maybe some other libs causing the trouble, I just found your lib among the libs with this issue

**Comment by hhff** [comment]
@Blackening999 - are you using the latest version of Ember Infinity?  We added the keys fallback in the last ~3 months I think

**Comment by hhff** [comment]
Closing this - I think @Blackening999 just has an old build of ember-infinity

**Comment by Blackening999** [comment]
hhff you were right

**Comment by hhff** [comment]
Nice :)
